examples/phi2/phi2-loop.py

```diff
@@ -0,0 +1,41 @@
+﻿import onnxruntime_genai as og
+
+print("Loading model...")
+model=og.Model("model", og.DeviceType.CPU)
+print("Model loaded")
+tokenizer=model.create_tokenizer()
+print("Tokenizer created")
+
+prompt = '''def print_prime(n):
+    """
+    Print all primes between 1 and n
+    """'''
+
+input_tokens = tokenizer.encode(prompt)
+
+params=og.search_params(model)
+params.max_length = 256
+params.input_ids = input_tokens
+
+generator=og.Generator(model, params)
```

Comment:
Should this one be uppercase still? (It's uppercase in the python.cpp also but others are lowercase)